app/livechat/server/lib/Contacts.js

```diff
@@ -0,0 +1,67 @@
+import _ from 'underscore';
+
+import {
+	LivechatVisitors,
+} from '../../../models';
+
+
+export const Contacts = {
+	saveContact({ _id, name, email, phone } = {}) {
```

Comment:
Why not using `Livechat.registerGuest` ?

Reply:
registerGuest is based on token, we don't use token here.
I've created the "Contacts" service level, so he is using `LivechatVisitors.insert` method.